src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {
+                new Truck(),
+                new Bulldozer(),
+                new Excavator()
+        };
+        for (Machine part : machines) {
+            part.doWork();
```

Comment:
The messages inside the 'doWork' method should be more informative. They should indicate what type of Machine is working right now, whether it's a Truck, Bulldozer, or Excavator.